### DISCUSSION TITLE
[User Error :P] Allow disabling default discussion types

### DISCUSSION THREAD

**Discussion by Column01** [community/community#5665] [Discussions] [answered]
I maintain a repo, and I'd like for people to submit feature requests via an Issue, but the default "Ideas" discussion type kinda makes it hard to get people to just use an issue. Can there be an option to disable default discussion types if we don't want to use them, but want discussions enabled?

**Comment by laymonage** [comment, marked as answer]
You can just delete the discussion category. See https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#deleting-a-category.

**Reply by Column01** [reply]
That doesn't work on default categories (or maybe it's because I'm not a full maintainer?)

**Reply by laymonage** [reply]
That's probably the case, because the action would also move existing discussions in the deleted category to another category.

Here's an example where I deleted the Ideas category: https://github.com/laymonage/discussions-playground/discussions

**Reply by Column01** [reply]
I see, thanks! I will bug the owner to make me a Maintainer so I can delete them (I'm the sole maintainer right now anyways and he was going to the other day but forgot to lol)
